internal/cli/atlas/security/ldap/save.go

```diff
@@ -96,6 +100,9 @@ func SaveBuilder() *cobra.Command {
 	cmd.Flags().StringVar(&opts.bindPassword, flag.BindPassword, "", usage.BindPassword)
 	cmd.Flags().StringVar(&opts.caCertificate, flag.CaCertificate, "", usage.CaCertificate)
 	cmd.Flags().StringVar(&opts.authzQueryTemplate, flag.AuthzQueryTemplate, "", usage.AuthzQueryTemplate)
+	cmd.Flags().StringVar(&opts.mappingMatch, flag.MappingMatch, "", usage.MappingMatch)
+	cmd.Flags().StringVar(&opts.mappingLdapQuery, flag.MappingLdapQuery, "", usage.MappingLdapQuery)
+	cmd.Flags().StringVar(&opts.mappingSubstitution, flag.MappingSubstitution, "", usage.MappingSubstitution)
```

Comment:
Please add these flags to this test https://github.com/ehershey/mongocli/blob/572f9736c1b462595b9dea25b28347f59d9ac975/internal/cli/atlas/security/ldap/save_test.go#L57

Reply:
We also have the LDAP e2e test that has to be updated https://github.com/ehershey/mongocli/blob/572f9736c1b462595b9dea25b28347f59d9ac975/e2e/atlas/ldap_test.go#L120

Reply:
This [comment](https://github.com/ehershey/mongocli/blob/572f9736c1b462595b9dea25b28347f59d9ac975/internal/cli/atlas/security/ldap/save.go#L79) should be updated

Reply:
Done, done and done. For the e2e test do you think we should add anything fancier like different combination of new parameters? Or a save call with all of them empty?

Reply:
I think the more tests we have the better but I am not going to block this PR for this